docs/using.rst

```diff
@@ -168,6 +169,10 @@ the UI, you can use the plugin to obtain a cert by specifying
 to copy and paste commands into another terminal session, which may
 be on a different computer.
 
+Additionally you can specify scripts to run before and after verification
```

Comment:
I think people might be confused by this before/after notion. It's true that the scripts run before/after boulder does its validation, but from the user's perspective these scripts actually perform the authentication/validation task, and clean up from it?